lib/ansible/module_utils/netapp.py

```diff
@@ -16,6 +17,13 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+import json
```

Comment:
If this module is going to be compatible with python-2.4 then we also need to fallback to simplejson here:
``` python

try:
    import json
except ImportError:
    import simplejson as json
```

Since ansible itself needs either json of simplejson to work correctly, that should be good enough.